app/Entry.js

```diff
@@ -28,6 +38,41 @@ import { SettingsScreen } from './views/Settings';
 
 const Stack = createStackNavigator();
 
+const fade = ({ current }) => ({ cardStyle: { opacity: current.progress } });
+
+const screenOptions = {
+  cardStyleInterpolator: CardStyleInterpolators.forHorizontalIOS,
+  cardStyle: {
+    backgroundColor: 'transparent', // prevent white flash on Android
+  },
+  headerShown: false,
+};
+
+const ExportStack = () => (
```

Comment:
This seems like it could go in Export.js as `getExportStack(Navigator, screenOptions)`, perhaps?

Happy to be overruled, it just seems weird to pull in all the steps and assemble them here, when it could be done there.

(And yes, maybe Onboarding dir should own it's stack)

Reply:
IMO it would be best to have a `Navigation` directory that handles constructing the `<Navigation/>` component for `App.js`. 

I think keeping the navigation creators together is useful so that we ensure we navigate between sections of the app consistently (e.g. with screen options), as navigator inheritance can be messy:

I think we should revisit this when we do the tab restructure, so it can be something like: 
```
navigation/
    index.js
    onboarding/
    mainTabs/
    export/
```

So that we can group construction under the `navigation` folder, and keep abstracted from views

Reply:
Ah, makes sense. Thanks for the thoughts